MinecraftClient/Mapping/World.cs

```diff
@@ -78,6 +89,30 @@ public class World
             }
         }
 
+        public World()
+        {
+            Location.world = this;
```

Comment:
This feels like a hack because two `World` instances cannot exist at the same time without breaking location calculation. That class was initially designed to allow multiple instances to coexist, however, since there will not be multiple worlds for now, how about moving this field to the `World` class, like `World.Current` to better follow the [singleton](https://www.tutorialspoint.com/design_pattern/singleton_pattern.htm) design pattern?